packages/KaSim/KaSim.4.0.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "1.2"
+name: "KaSim"
+version: "4.0"
```

Comment:
Version numbers should not be included in opam files - they are inferred from the parent directory name.